src/components/UsernameWithSuggestions.tsx

```
@@ -0,0 +1,70 @@
+import { ListItemText, MenuItem, TextField } from "@mui/material";
+import { WidgetProps } from "@rjsf/utils";
+import React from "react";
+interface UsernameWidgetProps {
+  formContext: any
+  value: any;
+  onChange: (value: any) => void;
```

ESLint

[error] 'value' is defined but never used.

(no-unused-vars)